src/classes/STG_InstallScript.cls

```diff
@@ -222,6 +232,22 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
+     /*******************************************************************************************************
+    * @description Disables Preferred Phone handling when upgrading to 1.81
+    * @return void
+    */
+    
+    global static void disablePreferredPhoneEnforcement(Version previousVersion) {
+        if(previousVersion != null && previousVersion.compareTo(new Version(1,81)) < 0) {
+            UTIL_Debug.debug('***Upgrading to version 1.81');
+            Hierarchy_Settings__c orgSettings = UTIL_CustomSettingsFacade.getSettings();
```

Comment:
@lifewithryan @skosgi1 I'm not sure you were looking at the most recent version of the code. We are using the custom settings facade here now.

Reply:
I was a little confused at standup because I remembered troubleshooting this with Samsheeth a week or two ago!

Reply:
I must be missing something in github then...I've long suspected I'm not seeing the latest things...no idea WHAT is happening but cool :)

Reply:
It happened to me once or twice over the past few weeks as well... I think it maybe has something to do with clicking on the PR link from the email of a past commit but I'm not sure.

Reply:
I think too many commits from my end added up to more confusion :)